### PR TITLE
Add sriov antrea CI scripts

### PR DESCRIPTION
**Pull request by abdallahyas**
Added the scripts to setup and test the antrea CI.

### CODE REVIEW

**Comment by abdallahyas** [comment]
@moshe010

File: sriov_antrea_install.sh [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+export TIMEOUT=${TIMEOUT:-300}
+export POLL_INTERVAL=${POLL_INTERVAL:-10}
+
+export ANTREA_CNI_REPO=${ANTREA_CNI_REPO:-https://github.com/vmware-tanzu/antrea.git}
+export ANTREA_CNI_BRANCH=${ANTREA_CNI_BRANCH:-master}
+export ANTREA_CNI_PR=${ANTREA_CNI_PR:-'786'}
+
+export SRIOV_CNI_REPO=${SRIOV_CNI_REPO:-https://github.com/intel/sriov-cni}
```
Comment (moshe010) [inline sriov_antrea_install.sh]: why you need sriov-cni?

**Review by moshe010 — CHANGES_REQUESTED** [review]

File: sriov_antrea_install.sh (hunk 1 above)
Comment (abdallahyas) [reply sriov_antrea_install.sh]: it was a left over from the copied file. will remove it now.

File: sriov_antrea_install.sh [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+export CNI_BIN_DIR=${CNI_BIN_DIR:-/opt/cni/bin/}
+export CNI_CONF_DIR=${CNI_CONF_DIR:-/etc/cni/net.d/}
+export KUBECONFIG=${KUBECONFIG:-/var/run/kubernetes/admin.kubeconfig}
+
+# generate random network
+N=$((1 + RANDOM % 128))
+export NETWORK=${NETWORK:-"192.168.$N"}
+
+#TODO add autodiscovering
+export MACVLAN_INTERFACE=${MACVLAN_INTERFACE:-eno1}
```
Comment (moshe010) [inline sriov_antrea_install.sh]: Why macvlan?

Comment (abdallahyas) [reply sriov_antrea_install.sh]: it is a leftover, and it is not used in the script. will go over the script to clean it up from any other leftovers or unused variables.

**Review by moshe010 — APPROVED** [review]

**Pull request merged**